doctests/home-json.js

```diff
@@ -0,0 +1,149 @@
+// EXAMPLE: js_home_json
+// REMOVE_START
+import assert from 'assert';
+// REMOVE_END
+// STEP_START import
+import {
+    createClient,
+    SchemaFieldTypes,
+    AggregateGroupByReducers,
+    AggregateSteps,
+} from 'redis';
+// STEP_END
+
+// STEP_START connect
+const client = await createClient();
+await client.connect();
+// STEP_END
+// REMOVE_START
+try {
+    await client.ft.dropIndex('idx:users', { DD: true });
+} catch{}
+
+await client.del('user:1', 'user:2', 'user:3');
```

Comment:
`FT.DROPINDEX idx:users DD` already delete the keys, why are we deleting them again?

Reply:
Often, there isn't an existing index (hence the try...catch as I mentioned in the previous reply), so presumably the keys don't get deleted in this case. However, keys like user:1, etc, are common in docs and test code. If user:1 exists but isn't a JSON key then you'll get the "wrong kind of value" error. If you think I'm being overly cautious here then I'll remove the client.del here, but I have seen this kind of thing happen occasionally.